README.md

```diff
@@ -0,0 +1,152 @@
+# bazel_rules_detekt
```

Comment:
I followed https://github.com/bazelbuild/rules_jvm_external

Originally I've written exactly what you suggested, but then realized that it'll be harder to remember the repo name (which is quite important in Bazel world since you download from GitHub releases directly)

Reply:
I see your motivation but the same name is in the... repo name. And in the URL.